framework/source/class/qx/lang/Number.js

```diff
@@ -74,6 +74,32 @@ qx.Class.define("qx.lang.Number",
       } else {
         return nr;
       }
+    },
+
+
+    /**
+     * Checks the equality of two numbers regarding the imprecision of floats.
+     *
+     * @param x {Number}
+     * @param y {Number}
+     * @return {Boolean}
+     */
+    equals : function(x, y)
```

Comment:
Sorry to be a pain ... but I think such a call should be called `almostEqual`

at least via 
https://www.google.ch/search?num=100&newwindow=1&dcr=0&q=almost+equal
you find  many other attempts to solve that problem

Reply:
I disagree. This is actually to check if two numbers are equal. It just tries to bypass the inefficiencies of a computer to precisely represent float numbers.

It becomes obvious for the example that 0.1 + 0.2 = 0.3. Both sides of that equation are equal, not "almost" equal.

btw: I'd really prefer if you put your criticism here at once and not in dribs and drabs.